files/templates/alertmanager/alertmanager.yaml

```diff
@@ -32,6 +32,8 @@ route:
   # Service Level slack -- chooses the slack channel based on the provider
   [[- if or (eq .Provider "aws") (eq .Provider "azure") ]]
   - receiver: team_phoenix_slack
+  [[- else if or (eq .Provider "capg") (eq .Provider "capa") (eq .Provider "capz") ]]
```

Comment:
I think it does not you need 2 or

Reply:
You can maybe use `has` instead https://stackoverflow.com/a/53027732 ?

Reply:
Not related to you but I think we need to simplify this, probably with some kind of loop around a list of teams

Reply:
so I locally tried putting this

```
[[- if or (eq .Provider "google") (eq .Provider "do") (eq .Provider "aws") (eq .Provider "azure") ]]
```

instead of current this:
```
[[- if or (eq .Provider "aws") (eq .Provider "azure") ]]
```

and the unit tests passed.

This changed `alertmanager.yaml` template file is being rendered in this unit test https://github.com/giantswarm/prometheus-meta-operator/blob/master/service/controller/resource/alerting/alertmanagerconfig/resource_test.go and the test specifies `aws` for the above `Provider` value, so it seems that using multiple operands works fine.

Reply:
> You can maybe use has instead

I would prefer to leave refactoring to Team Atlas, as I'm not very much familiar with the codebase :)

Reply:
I was thinking about us doing it :D